### ISSUE TITLE
Sushi Go Crafting

### ISSUE THREAD

**Issue by Buuz135** [EnigmaticaModpacks/Enigmatica9#190]
### URL

https://www.curseforge.com/minecraft/mc-mods/sushigocrafting

### Why would you like the mod added?

Its a good food mod

**Comment by Kanzaji** [comment]
Agreed, Sushi made with it is good, and I don't even normally eat a lot of sushi!

**Comment by NielsPilgaard** [comment]
Ah what the hell, why not 😄

**Comment by theboo** [comment]
Niller nooooo, you gotta say not until Itamae kitty is added!

**Comment by NielsPilgaard** [comment]
Boo you have to dm me a list of the stuff you're blackmailing Buuz for, otherwise I make mistakes like this :(
